app/views/chat_channels/show.json.jbuilder

```diff
@@ -1,4 +1,4 @@
-json.messages @chat_channel.messages.order("created_at DESC").limit(50).reverse do |message|
+json.messages @chat_messages.reverse do |message|
```

Comment:
I decided only to move the section of the collection that was talking to the DB out and leave the reverse here bc I think that gives some color as to what the view is doing to the DB information we just grabbed. Does that make sense?